lib/language_pack/installers/rbx_installer.rb

```diff
@@ -1,20 +0,0 @@
-require "language_pack/installers/ruby_installer"
-require "language_pack/shell_helpers"
-
-class LanguagePack::Installers::RbxInstaller
```

Comment:
Does dropping this file mean the buildpack is dropping rubinius support with this release? If so, it should be in the changelog. If not, maybe this removal belongs in another PR?